### ISSUE TITLE
Notebook hangs opening

### ISSUE THREAD

**Issue by sbatten** [microsoft/vscode#169876]
I was working in a notebook in a dev container. When I reopened the workspace locally and tried to open the notebook, it hangs forever with the below error.

<img width="842" alt="image" src="https://user-images.githubusercontent.com/6561887/209203178-df6894a6-affa-4a23-b423-f0c3b776de0d.png">

**Comment by rebornix** [comment]
The error is irrelevant to the file opening, we get that error as the text model is already synced but the editor failed to open due to webview failure.

**Comment by mjbvz** [comment]
@sbatten Are you still seeing this?

**Comment by sbatten** [comment]
No, tested with same notebook, it still hung for a few seconds, but then seems like some logic kicked in to recover.
